protocol/json-schemas/tbdex-message-schema.json

```diff
@@ -0,0 +1,276 @@
+{
+    "definitions": {
+        "metadata": {
+            "type": "object",
+            "properties": {
+                "id": {
+                    "type": "string"
+                },
+                "contextId": {
+                    "type": "string"
+                },
+                "from": {
+                    "type": "string"
+                },
+                "to": {
+                    "type": "string"
+                },
+                "createdTime": {
+                    "type": "number"
```

Comment:
should this be
```
type: string
format: date-time
```
?

```

Reply:
Yep, cool, didn't know that was a thing!

Reply:
Actually, these are longs/instants in the readme, which would be `number` instead of `datetime`. Following up in slack on whether they should be consistent types.